Components/Trips/Trips.js

```diff
@@ -184,6 +184,70 @@ const mockPreviews = [
 ];
 
 const Trips = ({navigation}) => {
+
+  let [reformattedTrips, setReformattedTrips] = useState([]);
+
+  const handleTripsFetch = () => {
+    getAllTrips()
+      .then(fetchedData => {
+        console.log(fetchedData);
+        reformatTripsData(fetchedData.data.allTrips);
+      })
+      .catch(error => {
+        console.log(error)
+      })
+  }
+
+  const findFinalDestination = (destinationSet) => {
+    let copiedData = [...destinationSet];
+    copiedData.reverse();
+    console.log(copiedData[0].destination.abbrev);
+    return copiedData[0].destination.abbrev
+  }
+
+  const findEndDate = (destinationSet) => {
+    let copiedData = [...destinationSet];
+    copiedData.reverse();
+    console.log(copiedData[0].endDate);
+    return copiedData[0].endDate
+  }
+
+  const findStartDate = (destinationSet) => {
+    let copiedData = [...destinationSet];
+    console.log(copiedData[0].startDate);
+    return copiedData[0].startDate
+  }
+
+  const reformatTripsData = (fetchedData) => {
+    let copiedTrips = [...fetchedData];
+    console.log(copiedTrips);
+      copiedTrips.forEach(trip => {
+        trip.finalDestinationAbbrev = findFinalDestination(trip.tripdestinationSet);
+        trip.startDate = findStartDate(trip.tripdestinationSet);
+        trip.endDate = findEndDate(trip.tripdestinationSet);
+        trip.description = 'Placeholder text'
+      })
+      console.log(copiedTrips);
+      setReformattedTrips(copiedTrips)
+  };
```

Comment:
This is huge! Thank you so much for cleaning this data to match our current component! It will certainly make things smoother.